docs/en/sql-reference/statements/explain.md

```diff
@@ -111,9 +111,11 @@ Dump query plan steps.
 
 Settings:
 
--  `header` — Prints output header for step. Default: 0.
--  `description` — Prints step description. Default: 1.
--  `actions` — Prints detailed information about step actions. Default: 0.
+-   `header` — Prints output header for step. Default: 0.
+-   `description` — Prints step description. Default: 1.
+-   `indexes` — Shows used indexes, the number of filtered parts, and granules for every index applied. Default: 0. Supported for [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md) tables.
```

Comment:
Correct statement is `the number of filtered parts and the number of filtered granules`.
And I don't know if it is the same :)